### PR TITLE
Add about section

### PR DESCRIPTION
**Pull request by juan-apa**
As a user I should be able to check the about section so I can learn more about Target MVD
[Trello card](https://trello.com/c/ImCuiXmQ)

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
   namespace 'api' do
     namespace 'v1' do
+      # resources :abouts, only: %i[:index :show], format: 'json'
```
Comment (santir489) [inline config/routes.rb]: Remove comments please

File: db/migrate/20191021133314_create_about.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateAbout < ActiveRecord::Migration[6.0]
+  def change
+    create_table :abouts do |t|
+      t.text :content, { null: false }
```
Comment (santir489) [inline db/migrate/20191021133314_create_about.rb]: I think braces are not necessary here

File: app/admin/abouts.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+ActiveAdmin.register About do
+  permit_params :content
+
+  scope :all, default: true
+
+  index do
+    column('Content', :content)
```
Comment (jpascual1994) [inline app/admin/abouts.rb]: I think you can do `column :content`

File: spec/models/about_spec.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+#  content    :text             not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+require 'rails_helper'
+
+describe About do
+  describe 'validations' do
+    subject { build :about }
```
Comment (jpascual1994) [inline spec/models/about_spec.rb]: Do you need this subject?

**Pull request merged**